tests/integration_tests/test_embeddings.py

```diff
@@ -248,3 +250,36 @@ async def run() -> CreateEmbeddingResponse:
     else:
         embeddings = await run()
         test.expected(embeddings)
+
+
+@pytest.mark.parametrize("model_id", [spec.deployment.value for spec in specs])
+@pytest.mark.parametrize(
+    "input",
+    itertools.product(["cat", "dog"], repeat=4),
+    ids=lambda x: "_".join(x),
+)
+async def test_multi_input_embeddings(
+    get_openai_client, model_id: str, input: List[str]
```

Comment:
Minor: `itertools.product` result is a tuple, not a list

Reply:
nice catch, fixed